package.json

```diff
@@ -20,10 +20,10 @@
   "author": "Jérôme Brunel",
   "license": "MIT",
   "dependencies": {
-    "glob": "^7.1.1",
+    "@vusion/webfonts-generator": "^0.5.5",
```

Comment:
Please update URLs pointing to the old `webfonts-generator` in the README :)

Reply:
URL updated!